templates/feed.php

```diff
@@ -12,7 +12,7 @@
 header( 'Content-Type: ' . feed_content_type( 'rss-http' ) . '; charset=' . get_option( 'blog_charset' ), true );
```

Comment:
Well I didn't because its not outputting it as such... I would say its up to the header func to handle this. Lets hold off now adding this and see what VIP say